lib/sunrise/sunrise.php

```diff
@@ -85,7 +85,7 @@ function handle_not_found_error( $error_type ) {
 /**
  * When provided, load a client's sunrise too
  */
-$client_sunrise = ABSPATH . '/vip-config/client-sunrise.php';
+$client_sunrise = ABSPATH . 'wp-content/vip-config/client-sunrise.php';
```

Comment:
This will break production sites since `vip-config` does not live under `wp-content` on containers.